pkg/disttask/framework/taskexecutor/task_executor.go

```diff
@@ -158,6 +162,30 @@ func (e *BaseTaskExecutor) checkBalanceSubtask(ctx context.Context) {
 	}
 }
 
+func (e *BaseTaskExecutor) updateSubtaskSummaryLoop(
+	checkCtx, runStepCtx context.Context, stepExec execute.StepExecutor) {
+	taskMgr := e.taskTable.(*storage.TaskManager)
+	ticker := time.NewTicker(updateSubtaskSummaryInterval)
+	defer ticker.Stop()
+	curSubtaskID := e.currSubtaskID.Load()
+	update := func() {
+		summary := stepExec.RealtimeSummary()
+		err := taskMgr.UpdateSubtaskRowCount(runStepCtx, curSubtaskID, summary.RowCount)
```

Comment:
How we ensure the update success at least once？

Reply:
We cannot ensure if there are problems with `UpdateSubtaskRowCount`. Fortunately, since updating summary is not a critical mission, I think it is fine to have errors.